renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
```

Comment:
@tkrugg You don't want the Algolia configuration?

> https://github.com/algolia/renovate-config-algolia

Reply:
It’s has been updated here with the Algolia one https://github.com/algolia/search-insights.js/pull/78